src/subscriber.ts

```diff
@@ -184,6 +211,25 @@ export class Message {
     }
   }
 
+  /**
+   * Acknowledges the message, expecting a response (for exactly once subscriptions).
```

Comment:
exactly-once delivery

Reply:
I went ahead and just did several project-wide searches for this.